src/externalState/defaultConfig.tsx

```diff
@@ -0,0 +1,6 @@
+import isEqual from "lodash.isequal";
+import { MembraneClientConfig } from "./createNoContextMembraneClient";
+
+export const DEFAULT_MEMBRANE_CLIENT_CONFIG: MembraneClientConfig = {
```

Comment:
as you said, by adding Default config we still include isEqual in this repository. We should remove it